server/server-html.jsx

```diff
@@ -50,7 +50,7 @@ export default function ServerHTML({ body, assets, locale, title, meta }) {
             <link href="https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,600" rel="stylesheet" type="text/css" />
             { assets.style.map((href, idx) =>
                 <link href={href} key={idx} rel="stylesheet" type="text/css" />) }
-            <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en"></script>
+            <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en,Intl.~locale.ru,Intl.~locale.fr,Intl.~locale.es,Intl.~locale.it"></script>
```

Comment:
Please find another solution for this; We will be removing all non-essential 3P javascript resources from steemit.com soon.